svn/common.py

```diff
@@ -23,9 +23,9 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
 
         self.__type = type_
 
-    def run_command(self, subcommand, args, success_code=0, 
+    def run_command(self, subcommand, args, success_code=0,
                     return_stderr=False, combine=False, return_binary=False):
-# TODO(dustin): return_stderr is no longer implemented.
+        # TODO(dustin): return_stderr is no longer implemented.
```

Comment:
They're TODOs. It's best if they stand out.

On Sep 10, 2016 02:35, "Tushar Makkar" <notifications@github.com> wrote:

> In svn/common.py
> <https://github.com/dsoprea/PySvn/pull/48#discussion_r78272434>:
>
> >                      return_stderr=False, combine=False, return_binary=False):
> > -# TODO(dustin): return_stderr is no longer implemented.
> > +        # TODO(dustin): return_stderr is no longer implemented.
>
> I am undoing this but any specific reason for undoing.. It wasn't looking
> good and was unindented before.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dsoprea/PySvn/pull/48/files/aea4ba15e6d3633f001abea5fefeeac49ec05d64#r78272434>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AArraiu1f-p6L2_EpQ1nytZ3DzDnJONyks5qok-zgaJpZM4J5HJK>
> .
>